packages/ui-svelte/src/Nav.svelte

```diff
@@ -0,0 +1,88 @@
+<script>
+  import clsx from 'clsx'
+  import Link from './Link.svelte'
+  export let path = ''
+  const displayPath = ['Vercel Examples']
```

Comment:
This should be made reactive

```suggestion
  $: displayPath = ['Vercel Examples']
```